tasks/adjust_config.yml

```diff
@@ -32,41 +32,21 @@
           cmd: |-
             docker exec --user={{ nextcloud_uid }}:{{ nextcloud_gid }} {{ nextcloud_identifier }}-server php /var/www/html/occ {{ item.appType }} --type={{ item.type }} --value={{ item.value }} -- {{ item.appConfig }} {{ item.configName }}
       with_items:
-        - appType: "config:app:set"
-          appConfig: "previewgenerator"
-          configName: "squareSizes"
-          type: string
-          value: "{{ nextcloud_preview_squareSizes }}"
-        - appType: "config:app:set"
-          appConfig: "previewgenerator"
-          configName: "widthSizes"
-          type: string
-          value: "{{ nextcloud_preview_widthSizes }}"
-        - appType: "config:app:set"
-          appConfig: "previewgenerator"
-          configName: "heightSizes"
-          type: string
-          value: "{{ nextcloud_preview_heightSizes }}"
         - appType: "config:system:set"
           appConfig: ""
           configName: "preview_max_x"
-          type: int
-          value: "{{ nextcloud_preview_preview_max_x }}"
+          type: "{{ 'int' if nextcloud_preview_preview_max_x is defined else 'string' }}"
+          value: "{{ nextcloud_preview_preview_max_x | default('null') }}"
         - appType: "config:system:set"
           appConfig: ""
           configName: "preview_max_y"
-          type: int
-          value: "{{ nextcloud_preview_preview_max_y }}"
+          type: "{{ 'int' if nextcloud_preview_preview_max_y is defined else 'string' }}"
+          value: "{{ nextcloud_preview_preview_max_y | default('null') }}"
         - appType: "config:system:set"
           appConfig: ""
           configName: "jpeg_quality"
           type: int
           value: "{{ nextcloud_preview_system_jpeg_quality }}"
-        - appType: "config:app:set"
-          appConfig: "preview"
-          configName: "jpeg_quality"
-          type: string
-          value: "{{ nextcloud_preview_app_jpeg_quality }}"
```

Comment:
You have kept `config:system:set jpeg_quality`, but it [seems like](nextcloud_preview_system_jpeg_quality) it's `config:app:set jpeg_quality` that still remains (with a new default value of `80`, not `60`).

Reply:
I'm nut sure I understand what you mean here. Currently there is a single jpeg_quality setting that can be used in NC and the current version of the playbook sets this via `config:system:set jpeg_quality`, using the variable `nextcloud_preview_system_jpeg_quality`. The default value 80 is the same that is set in the default NC config.

Reply:
My link above was broken (fixed now).

It should have gone to [this documentation page](https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/previews_configuration.html#jpeg-quality-setting), which talks about `config:app:set preview jpeg_quality`, not `config:system:set preview jpeg_quality`.

Seems like with your changes done in this PR, you're using `config:system:set`.

Reply:
NIce catch, thank you. Fixed in 49a5ce9